smoke-tests/play/build.gradle

```
@@ -35,8 +35,10 @@ dependencies {
   implementation "com.typesafe.play:filters-helpers_$scalaVersion:$playVersion"
 }
 
+def targetJDK = project.hasProperty("targetJDK") ? project.targetJDK : 11
+
 jib {
-  from.image = "bellsoft/liberica-openjdk-alpine:11"
-  to.image = "docker.pkg.github.com/open-telemetry/opentelemetry-java-instrumentation/smoke-play"
+  from.image = "bellsoft/liberica-openjdk-alpine:$targetJDK"
+  to.image = "docker.pkg.github.com/open-telemetry/opentelemetry-java-instrumentation/smoke-play-$targetJDK"
```

just to make image name a little clearer
```suggestion
  to.image = "docker.pkg.github.com/open-telemetry/opentelemetry-java-instrumentation/smoke-play-jdk$targetJDK"
```